public/index.html

```diff
@@ -20,8 +20,10 @@
     -->
     <link href='https://api.mapbox.com/mapbox-gl-js/v1.2.0/mapbox-gl.css' rel='stylesheet' />
     <link href='https://api.mapbox.com/mapbox-assembly/mbx/v0.18.0/assembly.min.css' rel='stylesheet'>
-    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.39.1/mapbox-gl.css' rel='stylesheet' />
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.0/mapbox-gl.css' rel='stylesheet' />
```

Comment:
We can't mess with that tileset

Reply:
I used prettier extension to indent the code, and most likely that is the reason of double quotes.

Reply:
My updated tile is new one which is coming from the same mapbox, and I have tested the app to make sure that it is working.